org.audacityteam.Audacity.yaml

```diff
@@ -182,7 +180,7 @@ modules:
     buildsystem: cmake-ninja
     builddir: true
     config-opts:
-      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DCMAKE_BUILD_TYPE=Release
```

Comment:
why? This allow having debug info.

Reply:
Do you need this debug information? This will just increase the binary size and I think it is not really useful for a "released" app.

Reply:
Alright. I just reverted the change then.